packages/manager/modules/dedicated-housing/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@ovh-ux/manager-dedicated-housing",
```

Comment:
commit need to have a breaking change for new modules

Reply:
done, thx